--- a/doc/Makefile
+++ b/doc/Makefile
@@ -12,7 +12,7 @@ PYTHONPATH    =
 # Internal variables.
 PAPEROPT_a4     = -D latex_paper_size=a4
 PAPEROPT_letter = -D latex_paper_size=letter
-ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees -T -W --keep-going $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) -n .
+ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees -T --keep-going $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) -n .
 
 .PHONY: help clean html dirhtml singlehtml pickle json htmlhelp qthelp devhelp epub latex latexpdf text man changes linkcheck
 
